CHANGELOG.md

```diff
@@ -5,18 +5,55 @@ All notable changes to OpenGHG will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/openghg/openghg/compare/0.8.2...HEAD)
+## [Unreleased](https://github.com/openghg/openghg/compare/0.9.0...HEAD)
+
+## [0.9.0] - 2024-08-14
+
+### Added
+
+- In `get_obs_surface`, if `inlet` is passed a slice and multiple search results are found, they will be combined into a single `ObsData` object with a "inlet" data variable. [PR #1066](https://github.com/openghg/openghg/pull/1066)
+- Packaging and release documentation. [PR #961](https://github.com/openghg/openghg/pull/961)
+- Options to search metastore by "negative lookup" and by "test functions"; the latter is used to implement 
+  searching by a `slice` object to find a range of values - [PR #1064](https://github.com/openghg/openghg/pull/1064)
+- Code to combine multiple data objects - [PR #1063](https://github.com/openghg/openghg/pull/1063)
+- A new object store config file to allow customisation of metadata keys used to store data in unique Datasources - [PR #983](https://github.com/openghg/openghg/pull/983)
```

Comment:
As per Brendan's comment - this should be removed from the CHANGELOG.md as this PR hasn't been merged.
```suggestion
```